compiler-plugin/testData/api/allFunctions.kt

```diff
@@ -75,6 +75,28 @@
 // equals(kotlin.Any): kotlin.Boolean
 // hashCode(): kotlin.Int
 // toString(): kotlin.String
+// class: Sub
+// equals(kotlin.Any): kotlin.Boolean
+// foo(kotlin.String): kotlin.Unit
+// hashCode(): kotlin.Int
+// toString(): kotlin.String
+// class: SubAbstract
+// <init>(): SubAbstract
+// equals(kotlin.Any): kotlin.Boolean
+// foo(kotlin.String): kotlin.Unit
+// hashCode(): kotlin.Int
+// toString(): kotlin.String
+// class: Super
+// equals(kotlin.Any): kotlin.Boolean
+// foo(kotlin.String): kotlin.Unit
```

Comment:
It might be good to render this as `foo(kotlin.String...)` or `foo(vararg kotlin.String)` to indicate that value parameter is `vararg`?

Reply:
Good point